packages/core/usePrevious/demo.vue

```diff
@@ -0,0 +1,21 @@
+<script setup lang="ts">
+import { usePrevious } from '@vueuse/core'
+import { useCounter } from '@vueuse/shared'
+
+const { count, inc, dec } = useCounter(0)
+const prevCount = usePrevious(count)
+</script>
+
+<template>
+  <div>
+    <p>counter current value：{{ count }}</p>
+    <p>counter previous value：{{ prevCount }}</p>
+    <button @click="inc()">
+      increase
+    </button>
+    <button style="margin-left: 8px;" @click="dec()">
```

Comment:
The docs have the [unocss](https://uno.antfu.me/?s=ml-2) pre-installed, which is more concise.
```suggestion
    <button class="ml-2" @click="dec()">
```